src/main/java/driver/Driver.java

```diff
@@ -0,0 +1,101 @@
+package driver;
+import config.TestProperties;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.firefox.FirefoxOptions;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Driver {
+    static WebDriver driver;
+
+    private static final String HOST = TestProperties.get("host");
+    private static final String BROWSERNAME = TestProperties.get("browsername");
+
+    private static final String USERNAME = TestProperties.get("username");
+    private static final String ACCESSKEY = TestProperties.get("accessKey");
+    private static final String BUILD = TestProperties.get("build");
+    private static final String NAME = TestProperties.get("name");
+    private static final String URL2 = TestProperties.get("url2");
+
+
+    public static WebDriver getDriver() throws MalformedURLException, URISyntaxException {
```

Comment:
handle MalformedURLException, URISyntaxException here, do not redirect that to test